torchvision/transforms/transforms.py

```diff
@@ -342,20 +342,22 @@ def __init__(self, size, scale=(0.08, 1.0), ratio=(3. / 4., 4. / 3.), interpolat
         self.ratio = ratio
 
     @staticmethod
-    def get_params(img):
+    def get_params(img, scale, ratio):
```

Comment:
I think this commit sneaked in by mistake?

Reply:
yup, gone now :)